src/sage/matroids/oriented_matroids/oriented_matroid.py

```diff
@@ -0,0 +1,271 @@
+r"""
+Oriented matroids construction
+
+Theory
+======
+
+Oriented matroids are a generalization of directed graphs, central hyperplane
+arrangements, vector arrangements, and other mathematical objects. There are
+many cryptomorphic definitions of oriented matroids. Precise definitions
+for each type can be found in that class's directory.
+
+See :wikipedia:`Oriented_matroid` for more details.
+
+
+Built-in oriented matroids
+==========================
+
+The types of oriented matroids currently implemented into sage are:
+    - Circuit Oriented Matroids
```

Comment:
```suggestion

The types of oriented matroids currently implemented into Sage are:

    - Circuit Oriented Matroids
```
Would is also be good to link to the classes by
```
- :class:`Circuit Oriented Matroids
  <sage.matroids.oriented_matroids.circuit_oriented_matroid.CircuitOrientedMatroid>`
```
?